src/components/__tests__/Bullet.ts

```diff
@@ -228,17 +274,33 @@ describe.skip('expansion', () => {
       }),
     ])
 
-    const subthoughts = await findSubthoughts('a')
-    const bulletB = await findAllByLabelText(subthoughts[0], 'bullet')
-    userEvent.click(bulletB[0])
+    const initialState = exportContext(store.getState(), [HOME_TOKEN], 'text/plain')
 
-    const exported = exportContext(store.getState(), [HOME_TOKEN], 'text/plain')
-    expect(exported).toEqual(`- __ROOT__
+    // Find all bullet elements:
+    const bulletElements = await screen.findAllByLabelText('bullet')
+
+    // Click the second bullet directly:
+    await userEvent.click(bulletElements[1]) // Index 1 for the second bullet
+
+    // Wait for the state to update:
+    await waitFor(() => {
+      const newState = exportContext(store.getState(), [HOME_TOKEN], 'text/plain')
+
+      // Expect that the state has changed:
+      expect(newState).not.toEqual(initialState)
+
+      // Expect that the entire "pin" section is removed:
+      expect(newState).not.toContain('- =pin\n  - true')
```

Comment:
The component tests should actually be testing what is rendered to the DOM. We already have reducer tests that make assertions about the exported thoughts.